_resources/port1.0/group/haskell_cabal-1.0.tcl

```diff
@@ -80,6 +80,14 @@ default build.cmd           {${haskell_cabal.bin}}
 default build.target        {new-build}
 default build.args          {${haskell_cabal.default_args}}
 
+# use compiler ${configure.cc}
+# see https://github.com/haskell/cabal/issues/1325
+default build.env           {PATH=${workpath}/bin:$env(PATH)}
+post-extract {
+    xinstall -m 0755 -d ${workpath}/bin
```

Comment:
Since 0755 are the default permissions, you don't need to restate that here. 

`xinstall -d ${workpath}/bin
`

Reply:
Done thanks.